lib/kitchen/verifier/pester.rb

```diff
@@ -615,6 +617,13 @@ def ps_hash(obj, depth = 0)
         end
       end
 
+      def ps_environment(obj)
+        obj.map do |k, v|
+          "$env:#{k} = '#{v}'"
+        end
+          .join("\n")
```

Comment:
I think this would be clearer if we assigned to an intermediary variable and then returned the joined value at the end, rather than this unusual syntax of calling a method on the result of a `do` block.

```suggestion
        commands = obj.map do |k, v|
          "$env:#{k} = '#{v}'"
        end

        commands.join("\n")
```